### DISCUSSION TITLE
Menu is not returing the translated menu items title

### DISCUSSION THREAD

**Discussion by bitforcesrl** [Shopify/storefront-api-feedback#156] [Ideas & Requests]
Hi all,
we are working with Hydrogen and we are able to get back the translations for collections and products but not for the navigation (online store -> menu). 

Thank you

```
fragment MenuItem on MenuItem {
  id
  resourceId
  tags
  title
  type
  url
}

query layoutMenus($language: LanguageCode, $headerMenuHandle: String!, $footerMenuHandle: String!) @inContext(language: $language) {
  shop {
    name
  }
  headerMenu: menu(handle: $headerMenuHandle) {
    id
    items {
      ...MenuItem
      items {
        ...MenuItem
        items {
          ...MenuItem
        }
      }
    }
  }
  footerMenu: menu(handle: $footerMenuHandle) {
    id
    items {
      ...MenuItem
      items {
        ...MenuItem
      }
    }
  }
}
```

**Comment by vixdug** [comment]
Hi @bitforcesrl - Thanks for bringing this to our attention. Let me take this back to the team.

**Reply by bitforcesrl** [reply]
hi @vixdug we are closer to present our demo hydrogen website to some of our selected customers, we are a Plus agency with ±100 shopify clients in our portfolio. 
We've tryed today to run new queries but the content is not translated
This issue it's a big pain point. Could you please give us some feedback.

thanks a lot

**Reply by bitforcesrl** [reply]
@vixdug i can see that other devs are reporting this issue. We are currently discussing quotations for some hydrogen replatforming with some of our customers and this issue it's a huge blocking, any updates on this ? thanks

**Comment by frehner** [comment]
Received an issue in Hydrogen about this: https://github.com/Shopify/hydrogen/issues/2431

**Comment by frehner** [comment]
Another Hydrogen issue https://github.com/Shopify/hydrogen/issues/713
